### DISCUSSION TITLE
GPU fan cant be slowed down

### DISCUSSION THREAD

**Discussion by johnfcrane** [Rem0o/FanControl.Releases#2890] [Q&A]
Hi I installed Fan Control this morning, it worked well but only recognized my gpu, I done a refresh and now said that the sensors cant be found. It still show that the gpu fan 2 is running 2800 RPM but there is no way to slow it down, since the morning I tried to use afterburner GPU tweak argus or argos program to control fans, but nothing. It is now stuck at that rpm It shows up to sensor monitors, and in fan control too but not let me turn it off. Done bios reset reinstalling all the apps but it nothing.

**Comment by Rem0o** [comment]
I replied to you by mail.
